Assets/SEE/DataModel/DG/GraphSearch.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FuzzySharp;
+
+namespace SEE.DataModel.DG
+{
+    /// <summary>
+    /// Allows searching for nodes by their string representation.
```

Comment:
What does `string representation` mean precisely?

Reply:
The documentation was outdated, previously this used the `ToShortString` method. It has now been updated to refer to the `SourceName` attribute, which is what we actually filter by.